src-cpp/rdkafkacpp.h

```diff
@@ -1222,6 +1233,33 @@ class Queue {
    */
   static Queue *create (Handle *handle);
 
+  /**
+   * @brief Forward/re-route queue \p dst.
+   * If \p dst is \c NULL, the forwarding is removed.
+   *
+   * The internal refcounts for both queues are increased.
+   * 
+   * @remark RD_KAFKA_Q_F_FWD_APP is set on this queue regardless of whether
```

Comment:
This is an internal flag not exposed to the application so it shouldnt be mentioned.
Rephrase to just say what the results of this op is.